app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,25 @@
+class PasswordResetsController < ApplicationController
+  def show
+    user = User.find_by(token: params[:id])
+    
+    if user
+      @token = user.token
+    else
+      redirect_to expired_token_path unless user
```

Comment:
You don't need the `unless user` since the `if user` above won't let it reach this line if `user` is truthy.